src/workflow.ts

```diff
@@ -75,6 +94,9 @@ export interface WorkflowContext extends DBOSContext {
 
   sleepms(durationMS: number): Promise<void>;
   sleep(durationSec: number): Promise<void>;
+
+  getConfiguredClass<C extends InitConfigMethod>(cls: C): ConfiguredClass<C, Parameters<C['initConfiguration']>[1]>;
```

Comment:
Same question here: can we remove the parameter because it should only return itself?

Reply:
It doesn't know what "itself" is.  That type is not retained here, it has to be told one way or another.  Otherwise we'd have to go through and try to templatize WorkflowContext on the class containing the function, but that sounds sort of awful.